include/libassert/stringification.hpp

```diff
@@ -28,6 +28,7 @@
 
 #if LIBASSERT_STD_VER >= 20
  #include <compare>
+ #include <format>
```

Comment:
Maybe something like this would be worthwhile since a few compilers took a while to support `<format>`
```cpp
#if defined(__has_include) && __has_include(<format>)
 #include <format>
 #define LIBASSERT_STD_FORMAT
#endif
```

Reply:
Looks like we had a race condition? Lemme know if you'd prefer I change to what you've suggested here.

Reply:
Your change looks good, thanks!